sgrep/tests/go/equivalence_naming_import_1.sgrep

```diff
@@ -0,0 +1 @@
+html.template.$FUNC(...)
```

Comment:
Should this be a .yaml file not .sgrep

Reply:
No. For sgrep repo testing, the patterns are in .sgrep extension as the test harness (thing that is run when you do "make test") uses files with .sgrep. https://github.com/returntocorp/sgrep/blob/develop/sgrep/tests/test.ml#L48

Reply:
For sgrep-rules repo, the patterns are in .yaml or .sgrep format.